lib/puppet/provider/cisco_interface/cisco.rb

```diff
@@ -206,8 +206,10 @@ def self.instances
     interfaces = []
     Cisco::Interface.interfaces.each do |interface_name, nu_obj|
       begin
-        # Not allowed to create an interface for mgmt0 or MgmtEth0/*
-        next if interface_name.match(/mgmt/i)
+        # Some interfaces cannot or should not be managed by this type.
+        # - Management Interface
+        # - NVE Interfaces (managed by cisco_vxlan_vtep type)
+        next if interface_name.match(/mgmt|nve/i)
```

Comment:
I'm thinking we may want to change this to allowing mgmt interfaces here, but forbid being able to flush them.

That way we can display the mgmt interface info with `puppet resource` but not modify them.

Reply:
@shermdog That seems reasonable.  I will make the change.

Reply:
I am going to make this change in a subsequent PR.  Will merge this PR without further modifications.